components/automate-dex/web/templates/password.html

```diff
@@ -29,6 +29,11 @@
       <span class="dex-error-text">Username or password is incorrect.</span>
     </div>
     {{ end }}
+    {{ if eq "something-else" "local" }}
```

Comment:
idk how to do this, @srenatus hopefully you can fix this. the styles are there for the h1. thank you!!